kolibri/core/assets/src/mixins/taskStrings.js

```diff
@@ -69,7 +69,7 @@ const taskStrings = createTranslator('TaskStrings', {
     context: 'Sync task status',
   },
   syncFacilityTaskLabel: {
-    message: "Sync '{facilityName}'",
+    message: 'Sync {facilityName}',
```

Comment:
This change and similar ones looks incorrect to me. From [the design system](https://design-system.learningequality.org/writing/#notes):

> Always wrap user-generated text in 'single quotes'.

Reply:
`facilityName` is interpolated to something like `'My facility' (a1b2)`, so the full message should match what you're saying. Check out the tests.

Reply:
Ah, interesting – I didn't realize that `facilityNameWithId` was used inside other strings. I think that this pattern, while DRY, is technically incorrect for i18n strings because we're essentially using string concatenation to join two incomplete phrases.

A more technically correct string would include both inputs:

```
     message: `Sync '{facilityName}' ({id})`, 
```

My understanding is that this less-DRY format is easier for the translators to work with effectively.

(cc @radinamatic in case you have guidance and @jredrejo from the original change)

None of this is super critical – just wanted to flag for discussion and future work.